cfgov/core/utils.py

```diff
@@ -128,6 +134,9 @@ def add_link_markup(tag):
         tag.contents.append(BeautifulSoup(svg_icon(icon), 'html.parser'))
         return str(tag)
 
+    elif is_image_tag(tag):
```

Comment:
I might be misreading this but, as written, this seems like it won't properly skip adding an icon to `<a href="external or download"><img></a>` links, because the `elif` here is against the `if icon` check, not the other `elif`s above. See my note about the test above: what is the desired behavior here?

Could you also update the `add_link_markup` docstring above (and maybe add a one-line comment here) to explain why this branch is here?

Reply:
Ok, that's fixed in c654196.